src/net_processing.cpp

```diff
@@ -2533,7 +2532,7 @@ bool PeerManagerImpl::CheckHeadersPoW(const std::vector<CBlockHeader>& headers,
 
     // Are these headers connected to each other?
     if (!CheckHeadersAreContinuous(headers)) {
-        Misbehaving(peer, 20, "non-continuous headers sequence");
+        Misbehaving(peer, 100, "non-continuous headers sequence");
```

Comment:
I don't know why we ever tolerated this, but it's been that way since headers-first was added in #4468. Perhaps to avoid a network split in the case where headers-first had an implementation bug?

Note that the `getheaders` message is older, see https://github.com/bitcoin/bitcoin/pull/4468/commits/341735eb8f42e898cf9d4d130709471e5d01abe2#r17027597